src/rtc.c

```diff
@@ -711,6 +713,53 @@ void RTC_Alarm_IRQHandler(void)
   HAL_RTC_AlarmIRQHandler(&RtcHandle);
 }
 
+/**
+  * @brief Attach Seconds interrupt callback.
+  * @param func: pointer to the callback
+  * @retval None
+  */
+void attachSecondsIrqCallback(voidCallbackPtr func)
+{
+  RTCSecondsIrqCallback = func;
+  HAL_RTCEx_SetSecond_IT(&RtcHandle);
+  HAL_NVIC_EnableIRQ(RTC_IRQn);
```

Comment:
The HAL_RTCEx_SetSecond_IT() function does not clear the SECF flag;
this could be done by `__HAL_RTC_SECOND_CLEAR_FLAG()`